src/main/java/org/isf/generaldata/ConfigurationProperties.java

```diff
@@ -66,7 +66,7 @@ protected ConfigurationProperties(String fileProperties, boolean exitOnFail) {
 	 * @param fileProperties - the file name (to be available in the classpath)
 	 * @param logger - the {@link Logger} of the concrete class
 	 */
-	public static final Properties loadPropertiesFile(String fileProperties, Logger logger) {
+	public static Properties loadPropertiesFile(String fileProperties, Logger logger) {
```

Comment:
this method is public, was final meant to be removed as well?

Reply:
@mwithi yes it it to be removed.  Updated the text of the PR to reflect the fact that it isn't limited to just `private` methods.